gatsby-node.js

```diff
@@ -10,10 +10,109 @@ const moment = require('moment');
 
 const events = require('./src/data/events/events');
 const highlightedEvents = require('./src/data/events/highlights');
+// const { TagFacesSharp } = require('@material-ui/icons');
```

Comment:
we can remove it right?

Reply:
I don't know what it does lmao. Eslint caught it as an unused variable warning so I commented it out.